SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -671,3 +694,18 @@ void UGlobalStateManager::SetDeploymentMapURL(const FString& MapURL)
 	UE_LOG(LogGlobalStateManager, Log, TEXT("Setting DeploymentMapURL: %s"), *MapURL);
 	DeploymentMapURL = MapURL;
 }
+
+
+void UGlobalStateManager::IncrementSessionIDAndUpdateSpatial()
```

Comment:
```suggestion
void UGlobalStateManager::IncrementSessionID()
```
Don't think there's a need to add "AndUpdateSpatial" as we assume that updating the GSM object locally implies updating the entity in spatial in other methods.

Reply:
Fight with @Vatyx

Reply:
I agree with Josh.

Reply:
Not yet done.